src/main/java/com/eternalcode/combat/fight/controller/FightTagController.java

```diff
@@ -56,17 +57,21 @@ void onEntityDamageByEntity(EntityDamageByEntityEvent event) {
 
         UUID attackedUniqueId = attackedPlayerByPerson.getUniqueId();
         UUID personToAddCombatTimeUniqueId = personToAddCombatTime.getUniqueId();
+        boolean isTaggedAttacker = this.fightManager.tag(attackedUniqueId, combatTime, CauseOfTag.PLAYER);
+        boolean isTaggedPerson = this.fightManager.tag(personToAddCombatTimeUniqueId, combatTime, CauseOfTag.PLAYER);
 
-        if (!this.fightManager.isInCombat(attackedUniqueId)) {
-            this.announcer.sendMessage(personToAddCombatTime, this.config.messages.playerTagged);
-        }
 
-        if (!this.fightManager.isInCombat(personToAddCombatTimeUniqueId)) {
-            this.announcer.sendMessage(attackedPlayerByPerson, this.config.messages.playerTagged);
+        if (isTaggedAttacker) {
+            if (!this.fightManager.isInCombat(attackedUniqueId)) {
```

Comment:
This is unnecessary - The `fightManager#tag` method returns whether the player has been tagged, so you don't have to check whether he is in combat again.
```suggestion
```